src/main/java/nl/ordina/jobcrawler/controller/MylerVacancyScraper.java

```diff
@@ -0,0 +1,91 @@
+package nl.ordina.jobcrawler.controller;
+
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.model.VacancyURLs;
+import nl.ordina.jobcrawler.service.ConnectionDocumentService;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+MylerVacancyScraper.java takes care of scraping vacancies from myler.nl
+At this moment getVacancies() returns a list with Vacancies that only exist of a vacancyURL and the broker. No details available (yet).
+ */
+
+@Component
+public class MylerVacancyScraper extends VacancyScraper {
+
+    private static final String BROKER = "Myler";
+    private static final String SEARCH_URL = "https://www.myler.nl/opdrachten/?search=" + SEARCH_TERM;
+
+    private List<Vacancy> vacancies = new ArrayList<>();
+
+    private final ConnectionDocumentService connectionDocumentService;
+
+    @Autowired
+    public MylerVacancyScraper(ConnectionDocumentService connectionDocumentService) {
+        this.connectionDocumentService = connectionDocumentService;
+    }
+
+    @Override
+    public List<Vacancy> getVacancies() throws IOException {
```

Comment:
Wat gebeurt er als deze method twee keer achter elkaar wordt aangeroepen?

Reply:
Het initiëren van de List is verplaatst naar de method body om een probleem te voorkomen, zie https://github.com/mostafa-221/jobcrawler-backend/commit/d07d302f69fdef151383e4a3c1b6b49a752f0064